models/modules/unet_generator_attn/unet_generator_attn_vid.py

```diff
@@ -379,7 +379,7 @@ def __init__(
         attention_block_types=("Temporal_Self", "Temporal_Self"),
         cross_frame_attention_mode=None,
         temporal_position_encoding=False,
-        temporal_position_encoding_max_len=25,
+        temporal_position_encoding_max_len=None,
```

Comment:
Keep the default 25 value because None cannot work, and should not be used for init.